### PR TITLE
feat: add additional popover placements

### PR DESCRIPTION
**Pull request by nmerget**
## Proposed changes

Add additional placements for popover and tooltip.

## Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Refactoring (fix on existing components or architectural decisions)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: packages/components/src/components/popover/popover.spec.tsx [hunk 1]
```diff
@@ -31,14 +31,3 @@ test.describe('DBPopover', () => {
...
-test.describe('DBPopover', () => {
-	test('should not have any A11y issues', async ({ page, mount }) => {
-		await mount(comp);
-		const accessibilityScanResults = await new AxeBuilder({ page })
-			.include('.db-popover')
-			.analyze();
-
-		expect(accessibilityScanResults.violations).toEqual([]);
-	});
-});
```
Comment (mfranzke) [inline packages/components/src/components/popover/popover.spec.tsx:44]: why do you remove this whole part (at least I wouldn't expected some change like this to happen within a PR like this)?

Comment (nmerget) [reply packages/components/src/components/popover/popover.spec.tsx:44]: I had issues with the local e2e testing.

I guess we had the issue here before as well, at the moment we can remove this test.

Comment (nmerget) [reply packages/components/src/components/popover/popover.spec.tsx:44]: https://github.com/db-ui/mono/actions/runs/6735460700/job/18308760379?pr=1765

The issue was there before

Comment (nmerget) [reply packages/components/src/components/popover/popover.spec.tsx:44]: https://github.com/db-ui/mono/pull/1767 <- own PR for this

**Review by mfranzke — APPROVED** [review]

**Pull request merged**